### PR TITLE
Fixes based on build warnings

### PR DESCRIPTION
**Pull request by arichiv**
This resolves all current compile errors except for those related to browser context. I need to followup in HTML for that.

***

### :boom: Error: connect ETIMEDOUT 173.230.149.95:443 :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jan 10, 2023, 12:47 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [CSS Spec Preprocessor](https://api.csswg.org/bikeshed/) - CSS Spec Preprocessor is the web service used to build Bikeshed specs.

:link: [Related URL](https://api.csswg.org/bikeshed/?url=https%3A%2F%2Fraw.githubusercontent.com%2Farichiv%2Fwebappsec-permissions-policy%2F48aa7288f08115a16e3484c87de91df9fe8e89dd%2Findex.bs&md-warning=not%20ready)

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/webappsec-permissions-policy%23498.)._
</details>

### CODE REVIEW

**Comment by arichiv** [comment]
All but browser context errors below are fixed now:

@arichiv ➜ /workspaces/webappsec-permissions-policy (main) $ bikeshed --print=plain -f spec
WARNING: The heading 'Changes since FPWD' needs a manually-specified ID.
WARNING: The following <var>s were only used once in the document:
  'declaration', in algorithm 'process-policy-attributes'
If these are not typos, please add an ignore='' attribute to the <var>.
WARNING: Multiple elements have the same ID 'default-allowlist'.
Deduping, but this ID may not be stable across revisions.
LINK ERROR: Multiple possible 'origins' dfn refs for '/'.
Arbitrarily chose https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-url-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#http-origin
  https://html.spec.whatwg.org/multipage/dom.html#concept-document-origin
  https://html.spec.whatwg.org/multipage/semantics.html#link-options-origin
  https://html.spec.whatwg.org/multipage/media.html#media-resource-origin
  https://html.spec.whatwg.org/multipage/origin.html#concept-origin
  https://html.spec.whatwg.org/multipage/origin.html#coop-enforcement-origin
  https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-origin
[=origins=]
LINK ERROR: No 'dfn' refs found for 'permissions-policy'.
<a data-link-type="dfn" data-lt="Permissions-Policy">Permissions-Policy</a>
LINK ERROR: Multiple possible 'origin' dfn refs for '/'.
Arbitrarily chose https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-url-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#http-origin
  https://html.spec.whatwg.org/multipage/dom.html#concept-document-origin
  https://html.spec.whatwg.org/multipage/semantics.html#link-options-origin
  https://html.spec.whatwg.org/multipage/media.html#media-resource-origin
  https://html.spec.whatwg.org/multipage/origin.html#concept-origin
  https://html.spec.whatwg.org/multipage/origin.html#coop-enforcement-origin
  https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-origin
<a data-link-type="dfn" data-lt="origin">origin</a>
LINK ERROR: Multiple possible 'origin' dfn refs for '/'.
Arbitrarily chose https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-request-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#concept-url-origin
  https://html.spec.whatwg.org/multipage/infrastructure.html#http-origin
  https://html.spec.whatwg.org/multipage/dom.html#concept-document-origin
  https://html.spec.whatwg.org/multipage/semantics.html#link-options-origin
  https://html.spec.whatwg.org/multipage/media.html#media-resource-origin
  https://html.spec.whatwg.org/multipage/origin.html#concept-origin
  https://html.spec.whatwg.org/multipage/origin.html#coop-enforcement-origin
  https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-origin
[=origin=]
LINK ERROR: No 'dfn' refs found for 'parse policy directive' that are marked for export.
<a data-link-type="dfn" data-lt="Parse policy directive">Parse policy
directive</a>
LINK ERROR: Multiple possible 'browsing context' dfn refs for '/'.
Arbitrarily chose https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:browsing context
  https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
  https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-browsing-context
<a data-link-type="dfn" data-lt="browsing context">browsing context</a>
LINK ERROR: Multiple possible 'browsing context' dfn refs for '/'.
Arbitrarily chose https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
The following refs show up multiple times in their spec, in a way that Bikeshed can't distinguish between. Either create a manual link, or ask the spec maintainer to add disambiguating attributes (usually a for='' attribute to all of them).
spec:html; type:dfn; for:/; text:browsing context
  https://html.spec.whatwg.org/multipage/browsers.html#browsing-context
  https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-params-browsing-context
<a data-link-type="dfn" data-lt="browsing context">browsing
    context</a>
LINE ~999: No 'dfn' refs found for 'generate and queue a report' with for='['reporting']'.
[=reporting/generate and queue a report=]
LINT: Unexported dfn that's not referenced locally - did you mean to export it?
<dfn data-dfn-type="dfn" id="serialized-permissions-policy" data-lt="serialized-permissions-policy" data-noexport="by-default">serialized-permissions-policy<a href="#serialized-permissions-policy" class="self-link"></a></dfn>
<dfn data-dfn-type="dfn" id="serialized-permissions-policy" data-lt="serialized-permissions-policy" data-noexport="by-default">serialized-permissions-policy<a href="#serialized-permissions-policy" class="self-link"></a></dfn>
 ✔  Successfully generated, with 8 linking errors

**Comment by arichiv** [comment]
@clelland have time to take a look?

File: index.bs [hunk 1]
```diff
@@ -308,7 +308,7 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
   <section>
     <h3 id="default-allowlists">Default Allowlists</h3>
     <p>Every <a>policy-controlled feature</a> has a <dfn export lt=
-    "default allowlist|default allowlists">default allowlist</dfn>. The
+    "default allowlist|default allowlists" for="/">default allowlist</dfn>. The
```
Comment (clelland) [inline index.bs:311]: Should this be `for="policy-controlled feature"` instead? It's not a top-level concept, but an attribute of `feature`

**Comment by clelland** [comment]
LGTM, thanks for the cleanup!

**Pull request merged**